Makefile

```diff
@@ -1,45 +1,49 @@
 # Replaces autotools for emscripten build.
 
+SERVE_IP   ?= localhost
+SERVE_PORT ?= 3010
+
 SHELL := bash
 BUILD_TAG := $(shell echo `git rev-parse --short HEAD`-`[[ -n $$(git status -s) ]] && echo 'dirty' || echo 'clean'` on `date --rfc-3339=seconds`)
 
 CC             := gcc
 CFLAGS         := -D_REENTRANT -DARCWEB -Wall -Werror -DBUILD_TAG="${BUILD_TAG}"
 CFLAGS_WASM    := -sUSE_ZLIB=1 -sUSE_SDL=2
-LINKFLAGS      := -lz -lSDL2 -sUSE_SDL=2 -lm -ldl
-LINKFLAGS_WASM := -sALLOW_MEMORY_GROWTH=1 -sFORCE_FILESYSTEM -sEXPORTED_RUNTIME_METHODS=[\"ccall\"] -lidbfs.js
-DATA           := ddnoise
+LINKFLAGS      := -lz -lSDL2 -lm -ldl
+LINKFLAGS_WASM := -sUSE_SDL=2 -sALLOW_MEMORY_GROWTH=1 -sFORCE_FILESYSTEM -sEXPORTED_RUNTIME_METHODS=[\"ccall\"] -lidbfs.js
+DATA           := ddnoise roms/riscos311/ros311 roms/arcrom_ext cmos arc.cfg
 ifdef DEBUG
   CFLAGS += -D_DEBUG -DDEBUG_LOG -g3
 #  --source-map-base not needed if .map is in the same dir as .wasm
   LINKFLAGS_WASM += -gsource-map
   BUILD_TAG +=  (DEBUG)
   $(info ❗BUILD_TAG="${BUILD_TAG}")
-  DATA += roms/riscos311/ros311 roms/arcrom_ext cmos arc.cfg
```

Comment:
When doing an emscripten build as part of archimedes live we don't want to include ROMs in the build package, as they're loaded dynamically. I know you had a minimal emscripten build working with the default emscripten UI. Maybe that could be a separate make target?

Reply:
Ah OK that must have broken somehow, and I couldn't work out why that wasn't the default :) Feel free to back it out, I was thinking about adding a bit more internal abstraction around loading support & config files anyway.